mps_youtube/config.py

```diff
@@ -67,6 +67,9 @@ def set(self, value):
 
         # handle known player not set
 
+        # sadly, allowed_values is faulty - assumes copy() is always
+        # accessible, causing exception on any string not among allowed_values
```

Comment:
Ah, now I see. I thought `.copy()` was added in python 3.0, but apparently it was added in python 3.3. d5874380d2be34103be7c019c472a3e16f456020 should fix it.